src/openrct2/drawing/Sprite.cpp

```diff
@@ -36,6 +36,91 @@
 using namespace OpenRCT2;
 using namespace OpenRCT2::Ui;
 
+static inline uint32 rctc_to_rct2_index(uint32 image)
+{
+    if      (                  image <  1575) return image;
+    else if (image >=  1607 && image <  4983) return image - 32;
+    else if (image >=  4986 && image < 17189) return image - 35;
+    else if (image >= 17191 && image < 18121) return image - 37;
+    else if (image >= 18123 && image < 23800) return image - 39;
+    else if (image >= 23804 && image < 24670) return image - 43;
+    else if (image >= 24674 && image < 28244) return image - 47;
+    else if (image >= 28246                 ) return image - 49;
+    else throw Exception(); // invalid RCT2 element
+}
+
+static void read_and_convert_gxdat(IStream * stream, size_t count, bool is_rctc, rct_g1_element *elements)
+{
+    auto g1Elements32 = stream->ReadArray<rct_g1_element_32bit>(count);
+    if (is_rctc) // RCTC element count
+    {
+        uint32 rctc = 0;
+        for (size_t i = 0; i < SPR_G1_END; ++i)
+        {
+            /* RCTC's g1.dat has a number of additional elements
```

Comment:
Block comments need `//`, rather than `/*`, as per the code style (yeah, I know it was like that, sorry)